jobs/integr8ly/w2-test-executor.yaml

```diff
@@ -0,0 +1,104 @@
+          default: 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
+          description: 'QE repository containing the tests source code.'
+      - string:
+          name: BRANCH
+          default: 'master'
+          description: 'Branch of the repository'
+      - string:
+          name: CLUSTER_URL
+          description: 'URL of cluster on which the W1 test will be executed.'
+      - string:
+          name: NAMESPACE_PREFIX
+          description: "Value used to prefix the names of the namespaces created during Integr8ly installation"
+      - string: 
+          name: ADMIN_USERNAME
+          default: 'admin@example.com'
+          description: 'Admin user name to login to Integreatly cluster.'
+      - string: 
+          name: ADMIN_PASSWORD
+          default: 'Password1'
+          description: 'Admin password to login to Integreatly cluster.'
+      - string:
+          name: EVALS_USERNAME
+          default: 'evals11@example.com'
+          description: 'Evals user name'
+      - string:
+          name: EVALS_PASSWORD
+          default: 'Password1'
+          description: 'Evals user password'
+      - string:
+          name: TIMEOUT_THRESHOLD
+          default: '1'
+          description: 'optionally increase timeout values. If you provide value 2 it means it will be sleeping/waiting two times longer'      
+    dsl: |
+        def err = null
+        try {
+            timeout(20) { 
+                ansiColor('gnome-terminal') { 
+                    timestamps {
+                        node('cirhos_rhel7') {        
+                            stage('Verify input') {
+                                if (!CLUSTER_URL) {
+                                    throw new hudson.AbortException('CLUSTER_URL parameter is required!')
+                                } // if
+                            } // stage
+
+                            stage('Run Launcher test') {
+                                buildStatus = build(job: 'launcher-tests', parameters: [
+                                    string(name: 'REPOSITORY', value: "${REPOSITORY}"),
+                                    string(name: 'BRANCH', value: "${BRANCH}"),
+                                    string(name: 'CLUSTER_URL', value: "${CLUSTER_URL}"),
+                                    string(name: 'NAMESPACE_PREFIX', value: "${NAMESPACE_PREFIX}"),
+                                    string(name: 'ADMIN_USERNAME', value: "${ADMIN_USERNAME}"),
+                                    string(name: 'ADMIN_PASSWORD', value: "${ADMIN_PASSWORD}"),
+                                    string(name: 'EVALS_USERNAME', value: "${EVALS_USERNAME}"),
+                                    string(name: 'EVALS_PASSWORD', value: "${EVALS_PASSWORD}"),
+                                    string(name: 'TIMEOUT_THRESHOLD', value: "${TIMEOUT_THRESHOLD}"),
+                                    booleanParam(name: 'CLEAN_RESOURCES', value: true)
```

Comment:
Should not this be false here? I thought it is there since the 3scale test expects some stuff to exist and this stuff is created by this Launcher test. So it should not clean the resources afterward. Or am I missing something?

Reply:
We are OK with `true` in this case IMO since "clean" means only whether to delete the newly created GH repo for the booster or not. We can go with `false` but it's just very minor difference IMO. 

3scale relies "only" on running booster in an OpenShift cluster (and we don't delete the booster after test)

Reply:
ah, ok, so even if the CLEAN_RESOURCES is set to true, it only removes underlying GH repository but it does not remve the booster itself. Ack, in that case 'true' can indeed be here.

Reply:
yes that's the case. We can extend this (to also delete the booster in OS) in the future.